Cargo.toml

```diff
@@ -36,3 +36,8 @@ tokio = { version = "1", features = ["rt-multi-thread", "macros"] }
 [profile.release]
 overflow-checks = true
 lto = true
+
+[features]
+default = ["openssl"]
```

Comment:
I found the following pointers:
- https://docs.rs/reqwest/latest/reqwest/tls/index.html
- [reqwest#2025 Consider switching to rustls as the default, and make openssl optional](https://github.com/seanmonstar/reqwest/issues/2025)
- [cloudflare-rs#133 Add rustls-tls feature to use rustls with reqwest](https://github.com/cloudflare/cloudflare-rs/pull/133/files)
- [minio-rs#61 add feature to switch native-tls, rustls-tls or disable tls at all](https://github.com/minio/minio-rs/pull/61)

@aalkhodiry I believe we should use the following `[features]` clause instead:
```
default = ["default-tls"]
default-tls = ["reqwest/default-tls"]
native-tls = ["reqwest/native-tls"]
rustls-tls = ["reqwest/rustls-tls"]
```